distributed/httpfile.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+import requests
```

Comment:
Or alternatively rewrite history on this branch to exclude these commits for now.